test/PowerShellEditorServices.Test/Console/InputPromptHandlerTests.cs

```diff
@@ -46,17 +47,17 @@ public void InputPromptHandlerReturnsValuesOfCorrectType()
                     CancellationToken.None);
 
             Assert.Equal(NameField, inputPromptHandler.LastField.Name);
-            inputPromptHandler.ReturnInputString(NameValue);
+            await inputPromptHandler.ReturnInputString(NameValue);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4993607)